src/data/kana/index.ts

```diff
@@ -1 +1,3 @@
-export { default as hiragana } from './hiragana';
+import { default as hiragana } from './hiragana';
+
+export { hiragana };
```

Comment:
I had to change the import/export logic here and it heavily confuses me. Is this some typescript shit?

Reply:
@cykler `export default {} from ...` is a proposal at stage 1 ([link](https://github.com/tc39/proposal-export-default-from)), so you can't count on its implementation. Maybe TypeScript doesn't support it yet but the default bundle settings for CRA did...?